addons/website_livechat/views/website_livechat.xml

```diff
@@ -37,6 +37,13 @@
                 <link rel="stylesheet" type="text/scss" href="/im_livechat/static/src/scss/im_livechat.scss"/>
             </xpath>
         </template>
+
+        <template id="assets_editor_inherit_website_livechat" inherit_id="website.assets_editor" name="website_livechat Assets Editor">
```

Comment:
Name those kind of template as the original one: `assets_editor`

Reply:
Just for the id or also for the name?

Reply:
Not done, to follow guideline.